packages/core/src/utils/styleUtils/ssrStyles.ts

```diff
@@ -102,9 +101,9 @@ export const detachExperienceStyles = (experience: Experience): string | undefin
 
     let currentNode: ComponentTreeNode | undefined = undefined;
 
-    const registeredComponenIds = Array.from(componentRegistry.values()).map(
-      ({ definition }) => definition.id,
-    );
+    // const registeredComponenIds = Array.from(componentRegistry.values()).map(
+    //   ({ definition }) => definition.id,
+    // );
```

Comment:
I guess we. can remove these comments if they are not needed anymore (there's one more below)